appveyor.yml

```diff
@@ -6,7 +6,7 @@ build:
 
 build_script:
  - cmake -P ci-build.cmake
- - cmake --build .
+ - cmake --build . --config Debug --target INSTALL
```

Comment:
I've reset `appveyor.yml` and `ci-build.cmake` to the originals, these were just to test while this PR was WIP